modules/7_ibmcloud/load_balancer.tf

```diff
@@ -20,26 +20,27 @@
 
 ################################################################
 ##### Network topology requirements
-##### Ref: https://docs.openshift.com/container-platform/4.7/installing/installing_platform_agnostic/installing-platform-agnostic.html
+##### Ref: https://docs.openshift.com/container-platform/4.14/installing/installing_platform_agnostic/installing-platform-agnostic.html
 ################################################################
 
 locals {
-  api_servers        = var.bootstrap_count == 0 ? var.master_ips : concat([var.bootstrap_ip], var.master_ips)
-  api_servers_count  = var.bootstrap_count + var.master_count
+  # Post Installation does not require `bootstrap` node to be in the LoadBalancer
```

Comment:
If I am not wrong (unless things changed recently), the bootstrap API server needs to be exposed as a temporary control plane for masters to connect during the installation.

Reply:
@prb112 Did you run this step to delete the bootstrap entries? https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/main/docs/quickstart.md#delete-bootstrap-node

This will remove bootstrap entries from the load balancer. Let me know if that is not happening.

Reply:
When masters boot, as defined in master.ign it will hit the api-int.<domain> entry with port 22623. Hence this is needed for masters to fetch the machine config from the machine config server running on bootstrap.

If your concern was with 6443 then I am not sure if it was used during shifting from temporary control plane to masters.